client/web/src/search/results/SearchAlert.tsx

```diff
@@ -26,7 +26,16 @@ export const SearchAlert: React.FunctionComponent<React.PropsWithChildren<Search
     <Alert className="my-2" data-testid="alert-container" variant="info">
         <H3>{alert.title}</H3>
 
-        {alert.description && <Markdown className="mb-3" dangerousInnerHTML={renderMarkdown(alert.description)} />}
+        {alert.description && (
+            <Markdown
+                className="mb-3"
+                dangerousInnerHTML={renderMarkdown(alert.description, {
+                    // Disable autolinks to rev specs are not rendered as email links
+                    // (e.g. sourcegraph@4.0.1)
```

Comment:
Suggestion to match our [style guide](https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/style_and_mechanics/#abbreviations-acronyms-latinisms-jargon) + make extra clear what the example is
```suggestion
                    // (for example, "sourcegraph@4.0.1")
```